spec/desks_spec.js

```diff
@@ -154,4 +158,66 @@ describe('desks', () => {
         // And clean up the desk we created
         desks.remove('Test Desk A');
     });
+
+    fit('can enforce incoming, outgoing and onstage rules', () => {
+        // Send stories go to incoming stage
+        desks.newDeskBtn.click();
+        desks.deskNameElement().sendKeys('Test Desk');
+        desks.deskSourceElement().sendKeys('Test Source A');
+        desks.setDeskType('authoring');
+        desks.actionSaveAndContinueOnGeneralTab();
+
+        desks.getNewStageButton().click();
+        desks.stageNameElement().sendKeys('Test Stage A');
+        desks.stageDescriptionElement().sendKeys('Test Desk A Stage A Description');
+        desks.setStageIncomingMacro('Validate for Publish');
+        desks.saveNewStage();
+
+        desks.getNewStageButton().click();
+        desks.stageNameElement().sendKeys('Test Stage B');
+        desks.stageDescriptionElement().sendKeys('Test Desk B Stage B Description');
+        desks.setStageOutgoingMacro('Validate for Publish');
+        desks.saveNewStage();
+
+        desks.getNewStageButton().click();
+        desks.stageNameElement().sendKeys('Test Stage C');
+        desks.stageDescriptionElement().sendKeys('Test Desk C Stage C Description');
+        desks.setStageMovedOntoMacro('Validate for Publish');
+        desks.saveNewStage();
+
+        desks.showTab('people');
+        desks.addUser('admin');
+        desks.close();
+
+        // confirm story is created on working stage
+        monitoring.openMonitoring();
+        browser.refresh();
+        workspace.selectDesk('Test Desk');
+        authoring.createTextItem();
+        authoring.writeTextToHeadline('new item');
+        authoring.save();
+        expect(monitoring.getGroupItems(0).count()).toBe(1);
+
+        // confirm incoming rule kicks in
+        authoring.sendTo('Test Desk', 'Test Stage A');
+        assertToastMsg('error', 'BODY_HTML is a required field');
```

Comment:
Thanks @naspeh for the notice. There was a `fit` left over in the changes. It should be good now.